Class3/selenium_website_links.py

```diff
@@ -0,0 +1,41 @@
+import time
+from selenium import webdriver
+
+# Function that prints the html tag of all the links (<a> tags) on the website. Default webpage is: https://www.inmar.com
+def extract_links(url="https://www.inmar.com"):
```

Comment:
Don't use inmar website please. There are many more out there

Reply:
My bad. Just focused on script pass/fail. Hence, went unnoticed. Removed inmar in all the places.